CMakeLists.txt

```diff
@@ -202,6 +202,10 @@ if (NOT SKIP_TESTS)
   set(SKIP_TESTS FALSE)
 endif()
 
+if (NOT SKIP_TOOLS)
```

Comment:
Is this check really necessary?

Reply:
I copied it from SKIP_TESTS. But since in omim_add_tool_subdirectory it also does `if (NOT SKIP_TOOLS)`, it is unnecessary.